discord-command.js

```diff
@@ -0,0 +1,22 @@
+/*
+    This class holds all the commands for Discord
+*/ 
+class DiscordCommands {
+    constructor() {             
+       
+    }
+    function setBotStatus(bot, status){
+        bot.user.setStatus(status);

```

Comment:
what's the bot object need to be passed through this function, how could we use this function, do we need to import an external library ?